contrib-writing-guide.md

```diff
@@ -32,7 +32,7 @@ The documentation is built with Docusaurus, which requires Node.js. We recommend
 
 ```bash
 brew install npm
-brew install yarn
+brew install yarn # make sure yarn v1.x is installed
```

Comment:
```suggestion
# note: we use [yarn classic](https://classic.yarnpkg.com/lang/en/) to build the ClickHouse docs.
brew install yarn # make sure yarn v1.x is installed
```

Reply:
I do not use a Mac or brew; when you do the `brew install yarn` does it allow you to pick a version? (I use Linux).

Reply:
Running `brew install yarn` gives us yarn v1, so it's fine.
I don't think using brew to install yarn is a popular way though - actually I use volta, which is a version manager of node and its surrounding stuff. But anyway it would be super helpful to mention that we depend on yarn v1 :)

Reply:
Great, thanks for this.  I will merge when it finishes the CI tests.